src/components/api_var.js

```diff
@@ -0,0 +1,39 @@
+const appId = 'qORDqXRczV8lhB5tZQjX';
+const api = `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/${appId}`;
+
+const saveBook = async ({
+  id, title, type, author,
+}) => {
+  const result = await fetch(`${api}/books`, {
+    method: 'POST',
+    body: JSON.stringify({
+      item_id: id, title, author, category: type,
+    }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((res) => res.status);
+  return result;
+};
+
+const loadBooks = async () => {
+  const result = await fetch(`${api}/books`).then((res) => res.json())
+    .then((response) => response);
+
+  const bookArr = Object.entries(result)
+    .map(([id, book]) => ({ ...book[0], id: Number(id), type: book[0].category }));
+  return bookArr;
+};
+
+const removeBook = async (id) => {
+  const result = await fetch(`${api}/books/${id}`, {
+    method: 'DELETE',
+    body: JSON.stringify({ item_id: id }),
+    headers: {
+      'Content-type': 'application/json; charset=utf-8',
+    },
+  }).then((res) => res.status);
+  return result;
+};
```

Comment:
- [ ] Great job on the project functionality but there is an issue, The remove book functionality is not working after adding a book if you remove it and then refresh the page the book is still on the page. Please make sure the remove book functionality works as expected.
- There is a problem with the id you are passing in this function, It's not a number.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/162433851-64744298-f8b6-4fb7-9ca5-7400c2334e9f.png)

Reply:
fixed! 😄